Documentation.app/Contents/MacOS/FP concepts.playground/Pages/Rank-N polymorphism.xcplaygroundpage/Contents.swift

```diff
@@ -52,21 +56,25 @@ func singletonArray<T>(_ v: T) -> [T] {
 
  For example, the function `randomWordGenerator` below returns a random word of length `n`. As you can see `n` is a random number that is generated once each time you execute this program. `randomWordGenerator` captures the value of `n`, which means that the strings returned by `randomWordGenerator` will all have the same length during the execution of the program. Each time you call `randomWordGenerator` you will get a different word, but all of them will have the same length. If you re-run the program a second time, `n` will probably get a different value and the length of the words returned by `randomWordGenerator` will be different.
 */
+// nef:begin:hidden
+let letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+// nef:end
 let n: Int = .random(in: 0 ..< 10)
+
 let randomWordGenerator: () -> String = { 
-    (0 ..< n).forEach { _ in 
-        .random(in: "a" ... "z") 
-    }
+    (0 ..< n).map { _ in
+        "\(letters.randomElement()!)"
```

Comment:
For the sake of clarity, I'd maintain this as it is, I think it's easier to see the example.
```suggestion
        .random(in: "a" ... "z") 
```

Reply:
I have changed the approach, to keep the code as we had previously.